### PR TITLE
Feature: Add search language selector

### PR DESCRIPTION
**Pull request by haddadzineddine**
image

### CODE REVIEW

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -15,11 +15,16 @@
 
 	<title><%if @title.nil?%><%=$ORG_SITE%><%else%><%="#{@title} | #{$ORG_SITE}"%><%end%></title>
 
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/lipis/flag-icons@6.9.2/css/flag-icons.min.css"/>
 	<link rel="shortcut icon" href="/fav.ico" type="image/x-icon" />
-
 	<!-- Fonts -->
 	<link href='//fonts.googleapis.com/css?family=Droid+Sans:400' rel='stylesheet' type='text/css'>
 
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.14/dist/css/bootstrap-select.min.css">
```
Comment (syphax-bouazzouni) [inline app/views/layouts/_header.html.erb:23]: don't import/use  bootstrap select

File: app/views/search/index.html.haml [hunk 2]
```diff
@@ -17,6 +17,28 @@
     -# Advanced search options
     %div#search_options{style: "display: none;"}
       %div.form-group.row
+        %div.col-sm-2.mb-4 Search language
+        %div.col-sm-10
+          %select#select-beast(autocomplete="off" placeholder="Select a person...")
```
Comment (syphax-bouazzouni) [inline app/views/search/index.html.haml:22]: do a component

File: app/views/search/index.html.haml [hunk 3]
```diff
@@ -28,7 +50,11 @@
           %div.form-check
             = check_box(:search, :include_views, class: "form-check-input")
             = label(:search, :include_views, t('search.ontology_views'), class: "form-check-label")
-      %div.form-group.row
+
```
Comment (syphax-bouazzouni) [inline app/views/search/index.html.haml:53]: remove white this useless spaces

File: config/environments/development.rb [hunk 4]
```diff
@@ -1,10 +1,7 @@
-require "active_support/core_ext/integer/time"
```
Comment (syphax-bouazzouni) [inline config/environments/development.rb:1]: don't change this file

File: package.json [hunk 5]
```diff
@@ -4,7 +4,9 @@
   "dependencies": {
     "@hotwired/stimulus": "^3.0.1",
     "@hotwired/turbo-rails": "^7.1.1",
+    "bootstrap-select": "^1.13.18",
```
Comment (syphax-bouazzouni) [inline package.json:7]: don't use bootstrap-select we already have tom-select

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]

**Comment by haddadzineddine** [comment]
image

File: app/views/layouts/_header.html.erb [hunk 6]
```diff
@@ -15,11 +15,14 @@
...
 	<title><%if @title.nil?%><%=$ORG_SITE%><%else%><%="#{@title} | #{$ORG_SITE}"%><%end%></title>
 
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/lipis/flag-icons@6.9.2/css/flag-icons.min.css"/>
 	<link rel="shortcut icon" href="/fav.ico" type="image/x-icon" />
-
 	<!-- Fonts -->
 	<link href='//fonts.googleapis.com/css?family=Droid+Sans:400' rel='stylesheet' type='text/css'>
 
+  <!-- Latest compiled and minified JavaScript -->
+  <script defer src="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.14/dist/js/bootstrap-select.min.js"></script>
```
Comment (syphax-bouazzouni) [inline app/views/layouts/_header.html.erb:24]: remove this

File: app/views/search/index.html.haml [hunk 7]
```diff
@@ -15,8 +15,24 @@
       = link_to(t('search.show_advanced_options'), "javascript:void(0)", id: "advanced_options", data: {text_swap: t('search.hide_advanced_options')}, class: "form-text")
 
     -# Advanced search options
-    %div#search_options{style: "display: none;"}
+    %div#search_options
       %div.form-group.row
+        %div.col-sm-2.mb-4 Search language
+        %div.col-sm-10
+          %select#search_language.mb-4.w-25{autocomplete: "off",
```
Comment (syphax-bouazzouni) [inline app/views/search/index.html.haml:22]: use the component SelectInput

File: app/views/layouts/_header.html.erb [hunk 8]
```diff
@@ -15,11 +15,14 @@
 
 	<title><%if @title.nil?%><%=$ORG_SITE%><%else%><%="#{@title} | #{$ORG_SITE}"%><%end%></title>
 
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/lipis/flag-icons@6.9.2/css/flag-icons.min.css"/>
```
Comment (syphax-bouazzouni) [inline app/views/layouts/_header.html.erb:18]: import in application.scss.css

File: app/components/select_input_component/select_input_component_controller.js [hunk 9]
```diff
@@ -3,17 +3,54 @@ import TomSelect from "tom-select"
...
+                        return `<div title="${escape(data.id)}">
+                                <span class="${escape(data.icon)}"></span>
+                                <span class="title ml-2">${escape(data.title)}</span>  
+                            </div>`;
+                    }
+                }
+            }
+
+
+        } else {
```
Comment (syphax-bouazzouni) [inline app/components/select_input_component/select_input_component_controller.js:41]: don't do if else

**Review by syphax-bouazzouni — APPROVED** [review]

**Pull request merged**